client/src/Movies/Movie.js

```diff
@@ -1,17 +1,22 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import MovieCard from './MovieCard'
+import SavedList from './SavedList'
 
 export default class Movie extends Component {
   constructor(props) {
     super(props);
     this.state = {
       movie: null
     };
+    
   }
 
+  
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = parseInt(this.props.match.params.id, 10)
```

Comment:
I prefer:
```suggestion
    const id = Number(this.props.match.params.id)
```
tomato, toMAto 🍅